clusters/build-clusters/01_cluster/assets/prom_prow_rules.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  annotations:
+    source: https://github.com/loodse/prow-dashboards/blob/e4af4c0ec879a0971591adcfc1e6f52c2ff015ee/prometheus-rules.yaml
+  name: prow-rules
+  namespace: openshift-monitoring
```

Comment:
why this has to go to `openshift-monitoring` instead of `prow-monitoring`

Reply:
There is no prow-monitoring on build01, it only exists on api.ci

Reply:
The data comes from the k8s prometheus, not ours, anyway